src/devices/Card/Mifare/MifareCard.cs

```diff
@@ -487,7 +523,17 @@ public AccessType BlockAccess(byte blockNumber, byte[] sectorData)
         /// <param name="accessSector">The access desired</param>
         /// <param name="accessTypes">An array of 3 AccessType determining access of each block</param>
         /// <returns>The 3 bytes encoding the rights</returns>
-        public (byte B6, byte B7, byte B8) EncodeSectorAndClockTailer(AccessSector accessSector, AccessType[] accessTypes)
+        /// This is a synonym of EncodeSectorAndBlockTailer (for backward compatibility)
+        public (byte B6, byte B7, byte B8) EncodeSectorAndClockTailer(AccessSector accessSector, AccessType[] accessTypes) =>
```

Comment:
Can you mark this one as obsolete? So we can remove it in a further release?

Reply:
Yes, now marked as obsolete.